.ci.yaml

```diff
@@ -566,6 +566,7 @@ targets:
       shard: customer_testing
       tags: >
         ["framework", "hostonly", "shard", "linux"]
+      test_timeout_secs: "2700" # Allows 45 minutes (up from 30 default)
```

Comment:
Is this only needed on Linux, or is it needed for mac and Windows as well?

Reply:
Flutter/devtools only runs on the Linux shard, so I only applied it there.

Reply:
I could see us wanting this to be consistent though across all shards, or it could be confusing later on down the line. I will adjust!